instrumentation/okhttp/okhttp-3.0/testing/src/main/java/io/opentelemetry/instrumentation/okhttp/v3_0/AbstractOkHttp3Test.java

```diff
@@ -180,7 +181,16 @@ public void onResponse(Call call, Response response) {
         trace -> {
           trace.hasSpansSatisfyingExactly(
               span -> span.hasName("parent").hasKind(SpanKind.INTERNAL).hasNoParent(),
-              span -> span.hasName("GET").hasKind(SpanKind.CLIENT).hasParent(trace.getSpan(0)),
```

Comment:
I think we can leave this as it was. The network attributes are already tested in AbstractHttpClientTest.

Reply:
@laurit done